src/DssDirectDepositHub.t.sol

```diff
@@ -358,7 +358,20 @@ contract DssDirectDepositHubTest is DSTest {
         assertTrue(d3mTestPool.accrued());
     }
 
-    function test_wind_limited_maxDeposit() public {
+    function test_wind_limited_by_maxDeposit() public {
+        _windSystem(); // winds to 50 * WAD
+        d3mTestPlan.file("targetAssets", 75 * WAD);
+        d3mTestPool.file("maxDepositAmount", 5 * WAD);
```

Comment:
Ok so I think I didn't understand it well in the beginning, the deposit amount is per tx? I thought it was maximum amount in total for one address. In that case, it should count whatever is already there, so in this case `55` instead of 5.

Reply:
`maxDeposit()` is exactly what the ERC-4626 defines. IE if I were to call `deposit(amount)` what is the maximum value that `amount` could be taking current state into account.

Reply:
Ok got it, just checked the EIP, all good.